src/models/searchservice/request/VerticalSearchRequest.ts

```diff
@@ -33,6 +33,8 @@ export interface VerticalSearchRequest {
   location?: LatLong,
   /** {@inheritDoc QueryTrigger} */
   queryTrigger?: QueryTrigger,
+  /** The session ID */
+  sessionId?: string,
```

Comment:
Can we add a little more to this description? The Yext API docs describe this as: 'UUID used to track session state when cookies are blocked.' Maybe we can write something similar such as 'Used to track session state when cookies are blocked.'?